### PR TITLE
Set up homeview for component calling as stated in previous homeview version and styling worked on

### PR DESCRIPTION
**Pull request by BryanVeith**

### CODE REVIEW

File: src/components/HomeView.vue [hunk 1]
```diff
@@ -1,24 +1,62 @@
...
+              :key="tab"
+              :class="['tab-button', { active: currentTab === tab }]"
+              @click="currentTab = tab"
+            >
+            {{ tab }}
+          </button> -->
+          <button @click="currentTab = 'IngredientFeed'">
+            <font-awesome-icon
+              icon="carrot"
+              style="margin-right: 10px; font-size: 20px"
```
Comment (nashirj) [inline src/components/HomeView.vue:18]: can we make a class for this instead of having it inline

File: src/components/HomeView.vue [hunk 2]
```diff
@@ -46,3 +84,50 @@ export default {
   },
 };
 </script>
+<!-- TODO: remove unnessesary comments -->
```
Comment (nashirj) [inline src/components/HomeView.vue:87]: all these todos need to be addressed or have an issue number added before we merge this pr

File: src/components/HomeView.vue [hunk 3]
```diff
@@ -46,3 +84,50 @@ export default {
...
+<!-- TODO: remove App.vue routing and routing in router.js for everything other than home-->
+<!-- TODO(#9): replace this icon with the ones we're using in the figma diagrams -->
+
+<style scoped>
+.container {
+  padding: 0 !important;
+  margin: 0 !important;
+  max-width: 100%;
+}
+.col-md-2,
```
Comment (nashirj) [inline src/components/HomeView.vue:99]: should be a space above this, ditto elsewhere

**Comment by BryanVeith** [comment]
resolved comments in newest commit of #15

File: src/components/HomeView.vue [hunk 4]
```diff
@@ -30,19 +48,59 @@ export default {
...
-    //       error.message ||
-    //       error.toString();
-    //   }
-    // );
   },
 };
 </script>
+<!-- TODO(#16): fix css to keep button active when clicked off, it was until it suddenly stopped -->
+<!-- TODO(#16): set script to have the inital tab (IngredientFeed) button active -->
+<!-- TODO(#17): remove App.vue routing and routing in router.js for everything other than home-->
```
Comment (nashirj) [inline src/components/HomeView.vue:56]: I would prefer you did this in this PR please; you can just remove the stuff from router.js and App.vue that links to recipes, ingredients, and shopping list and then delete this TODO

**Review by nashirj — APPROVED** [review]
Really nice work :)

**Pull request merged**